tests/test_dbt_config_validators.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+import unittest
+import json
+
+from data_diff.dbt_config_validators import (
+    ManifestJsonConfig,
+    RunResultsJsonConfig,
+)
+
+
+RUN_RESULTS_PATH = "tests/dbt_artifacts/run_results/"
+MANIFEST_PATH = "tests/dbt_artifacts/manifests/"
+
+
+class TestRunResultsJsonConfig(unittest.TestCase):
+    def test_run_results(self):
+        # https://docs.getdbt.com/reference/artifacts/run-results-json
+        versions = ["v4"]
+
+        for version in versions:
+            with self.subTest(version=version):
+                with open(Path(RUN_RESULTS_PATH, f"run_results_{version}.json"), "r", encoding="utf-8") as run_results:
+                    RunResultsJsonConfig.parse_obj(json.load(run_results))
+
+
+class TestManifestJsonConfig(unittest.TestCase):
+    def test_manifest(self):
+        # https://docs.getdbt.com/reference/artifacts/manifest-json
+        versions = ["v4", "v5", "v6", "v7", "v8", "v9", "v10", "v11"]
+
+        for version in versions:
+            with self.subTest(version=version):
+                with open(Path(MANIFEST_PATH, f"manifest_{version}.json"), "r", encoding="utf-8") as manifest:
+                    ManifestJsonConfig.parse_obj(json.load(manifest))
```

Comment:
non blocking: if we wanted to be very careful we could assert that important fields are not empty

Reply:
By default, important fields will be validated to be non-empty. We can revisit if we see breakage in the wild because of this.